design/README.md

```diff
@@ -0,0 +1,34 @@
+# Rook Enhancement Proposals
+
+An Rook Enhancement Proposal is a way to propose, communicate, and coordinate on new efforts for the Rook project.
+
+It is inspired from our experience with the Kubernetes Enhancement process where many of our community participants collaborate each day.
+
+**This process is evolving, but is mandatory for all enhancements beginning with release-1.4.**
+
+## Quick start
+
+1. Socialize an idea with others. Make sure others think the work is worth doing, and are willing to review design and code changes required.
+2. Follow the process outlined in the [enhancement template](design_template.md).
+
+## FAQs
+
+### How do I know if I need to write a design document?
+
+If you are working on adding a new storage provider or a feature to an exist storage provider then you should write a design document.
```

Comment:
What about a single list for this section? 

Create a design document if:
- Adding a new storage provider
- Adding a new CRD
- Adding a significant feature to an existing storage provider. If the design is simple enough to describe in a github issue, you likely don't need a full design doc.